tools/sst_dump_tool.cc

```diff
@@ -199,10 +199,12 @@ int SstFileReader::ShowAllCompressionSizes(size_t block_size) {
     if (CompressionTypeSupported(i.first)) {
       CompressionOptions compress_opt;
       std::string column_family_name;
+      int unknown_level = -1;
       TableBuilderOptions tb_opts(imoptions, ikc, &block_based_table_factories,
                                   i.first, compress_opt,
                                   nullptr /* compression_dict */,
-                                  false /* skip_filters */, column_family_name);
+                                  false /* skip_filters */, column_family_name,
+                                  unknown_level);
```

Comment:
What do you think about just passing -1 directly (without using a variable) and adding a comment /* unknown level */

Something like this
```
false /* skip_filters */, column_family_name,
-1 /* unknown level */);
```